ChocolateChipCookieRecipe.md

```diff
@@ -8,11 +8,12 @@
 - 1/2 tsp baking powder
 - 3 TBSP water
 - 1 cup + 2 TBSP all-purpose flour
+- 15 cups of chocolate chips 
```

Comment:
Could you make it 20 cups?

Reply:
That's great, I'm gonna make the change!